app/models/concerns/project_association.rb

```diff
@@ -90,7 +90,7 @@ def update_elasticsearch_data
       ['team_id', 'user_id', 'read', 'source_id', 'project_id', 'unmatched'].each do |fname|
         data[fname] = self.send(fname).to_i if self.send("saved_change_to_#{fname}?")
       end
-      ['archived', 'sources_count'].each do |fname|
+      ['archived'].each do |fname|
```

Comment:
Sawy, now it doesn't make sense to have an array and a loop with one item :sweat_smile:

Reply:
Fixed